geoarrow-types/src/geoarrow/types/constants.py

```diff
@@ -0,0 +1,118 @@
+from enum import Enum
+
+
+class Encoding(Enum):
+    """Constants for encoding type.
+
+    Examples
+    --------
+
+    >>> from geoarrow import types
+    >>> types.Encoding.GEOARROW
+    <Encoding.GEOARROW: 5>
+    """
+
+    #: Unknown or uninitialized encoding
+    UNKNOWN = 0
+    #: Well-known binary encoding with a maximum of 2GB of data per array chunk
+    WKB = 1
+    #: Well-known binary encoding
+    LARGE_WKB = 2
```

Comment:
Should we consider that as different encodings? Like the "geoarrow" encoding can also be backed by multiple physical types, I would consider just a "WKB" as the encoding, and binary or large_binary two possible physical storage types.

Of course, it probably depends on how someone would use this whether the one or the other makes more sense